### ISSUE TITLE
Control4 Climate

### ISSUE THREAD

**Issue by mcsidor** [lawtancool/hass-control4#4]
Hi, what is the currently best option to control the Control4 Thermostat? I have tried hass-control4, homeassistant-control4, but there is no Climate control. I have tried different repositories, based on itsfrosty 2way driver, but it is still giving me a message, there is no module for control4 climate and also, it is more than 2 years old. This add-on looks as the newest, but it doesnt have the climate or support multiple buttons from keypad dimmers or configurable keypads.

**Comment by mellis** [comment]
Yes, those things are not currently supported. I believe the other projects all require a custom control4 driver to be installed in the system as well.

**Comment by mcsidor** [comment]
I have my control4 dealer license, so I have no problem installing it on the C4 controller. If there is a driver at drivercentral.io, chowmainsoft.com, blackwiredesigns.com, etc. I am willing to pay for it.

**Comment by granvillebarker** [comment]
This should be able to be closed with my update, please let me know if you find issues.
